fe-library-app/src/components/BookList/BookCard/BookCard.tsx

```diff
@@ -25,9 +51,11 @@ const BookCard = (props: BookProp) => {
         </div>
       </div>
       <div className={styles.extra_func}>
-        <button>Modify</button>
-        <button>Delete</button>
+        <button className={styles.book_phone_button} onClick={handleModifyClick}>Modify</button>
+        <button className={styles.book_desktop_button} onClick={() => setShowModifyModule(true)}>Modify</button>
+        <button onClick={handleDeleteClick}>Delete</button>
       </div>
+      {showModifyModule && <AddModifyBookModal onFinish={props.onClick} id={props.book.Id.toString()} onHide={() => setShowModifyModule(false)}/>}
```

Comment:
Split into multiple lines for the better overview

Reply:
Understand